qir/qat/Apps/Qat/Qat.cpp

```diff
@@ -374,3 +381,4 @@ int main(int argc, char** argv)
 
     return ret;
 }
+// Some minor update to qat
```

Comment:
I typically don't see the comments in the end of the file. Feels like a code fragment has been removed after this comment.

Reply:
```suggestion
```

Reply:
This is clearly a mistake from when I was testing.